forge-gui/res/cardsfolder/m/moonmist.txt

```diff
@@ -1,9 +1,9 @@
 Name:Moonmist
 ManaCost:1 G
 Types:Instant
-A:SP$ SetState | Cost$ 1 G | Defined$ Valid Creature.Human+DoubleFaced | Mode$ Transform | SubAbility$ DBCurseNonWolves | StackDescription$ SpellDescription | SpellDescription$ Transform all Humans. Prevent all combat damage that would be dealt this turn by creatures other than Werewolves and Wolves. (Only double-faced cards can be transformed.)
-SVar:DBCurseNonWolves:DB$ Effect | Name$ Moonmist Effect | ReplacementEffects$ RPrevent
+A:SP$ SetState | Defined$ Valid Creature.Human+CanTransform | Mode$ Transform | SubAbility$ DBCurseNonWolves | StackDescription$ SpellDescription | SpellDescription$ Transform all Humans. Prevent all combat damage that would be dealt this turn by creatures other than Werewolves and Wolves. (Only double-faced cards can be transformed.)
```

Comment:
the CanTransform there shouldn't be needed because it should be checked in the SetState Effect anyway